package.json

```diff
@@ -7,35 +7,36 @@
     "node": ">=20.0.0"
   },
   "scripts": {
-    "test": "turbo run test",
-    "test:coverage": "turbo run test:coverage",
-    "test:perf": "turbo run test:perf",
-    "test:quality": "turbo run test:quality",
-    "test:packages": "turbo run test --filter=./packages/*",
-    "test:coverage:packages": "turbo run test:coverage --filter=./packages/*",
-    "test:coverage:report": "node scripts/merge-coverage.js",
-    "test:integration": "turbo run test:integration",
+    "test": "wireit",
+    "test:coverage": "wireit",
+    "test:perf": "wireit",
+    "test:quality": "wireit",
+    "test:packages": "npm run test --workspaces --if-present",
+    "test:coverage:packages": "npm run test:coverage --workspaces --if-present",
+    "test:coverage:report": "wireit",
+    "test:integration": "npm run test:integration --workspaces --if-present",
     "test:web": "node scripts/test-web-ext.js web",
-    "test:e2e": "turbo run test --filter=e2e-tests",
-    "test:e2e:debug": "turbo run test:debug --filter=e2e-tests",
-    "test:e2e:server": "turbo run server --filter=e2e-tests",
-    "test:e2e:visual": "turbo run test:visual --filter=e2e-tests",
-    "lint": "turbo run lint",
-    "lint:fix": "turbo run lint:fix",
+    "test:e2e": "npm run test --workspace=e2e-tests",
+    "test:e2e:debug": "npm run test:debug --workspace=e2e-tests",
+    "test:e2e:server": "npm run server --workspace=e2e-tests",
+    "test:e2e:visual": "npm run test:visual --workspace=e2e-tests",
+    "lint": "wireit",
+    "lint:fix": "wireit",
     "knip": "knip",
     "knip:fix": "knip --fix",
-    "compile": "turbo run compile",
-    "precompile": "turbo run precompile",
+    "compile": "wireit",
+    "precompile": "wireit",
```

Comment:
hmm, if each package's compile step includes whatever precompile it needs, why do you need a repo-level pre-compile?